src/alpaca_eval/leaderboards/data_AlpacaEval/alpaca_eval_gpt4_leaderboard.csv

```diff
@@ -1,81 +1,84 @@
 ,win_rate,standard_error,n_wins,n_wins_base,n_draws,n_total,mode,avg_length
```

Comment:
this is very strange, it seems that it changed all the integers to floats for the metric. That used to be the case for an old `alpaca_eval` version, which one are you using?

Reply:
if it's not the latest can you just rerun the same command with the updated `alpaca_eval`? it should just reuse your cache (assuming you are on the same laptop/server) so it won't cost anything, it's just a formating question.

Reply:
Hi @YannDubs , I have fixed this formatting issue of the `leaderboard.csv`. Now the `avg_length` entries are all integers instead a float. Thanks for pointing out the issue and help